e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -321,29 +347,246 @@ describe("scenarios > search", () => {
+        cy.visit("/");
+
+        getSearchBar().clear().type("reviews{enter}");
+        cy.wait("@search");
+
+        expectSearchResultItemNameContent({
+          itemNames: [
+            NORMAL_USER_TEST_QUESTION.name,
+            ADMIN_TEST_QUESTION.name,
+            REVIEWS_TABLE_NAME,
+          ],
+        });
+
+        cy.findByTestId("created_by-search-filter").click();
+
+        popover().within(() => {
+          cy.findByText("Robert Tableton").click();
+          cy.findByText("Bobby Tables").click();
+          cy.findByText("Apply filters").click();
+        });
+        cy.url().should("contain", "created_by");
+
+        expectSearchResultItemNameContent({
+          itemNames: [NORMAL_USER_TEST_QUESTION.name, ADMIN_TEST_QUESTION.name],
+        });
+      });
+
+      it("should be able to remove a user from the `created_by` filter", () => {
+        cy.visit(
+          `/search?q=reviews&created_by=${NORMAL_USER_ID}&created_by=${ADMIN_USER_ID}`,
+        );
+
+        cy.wait("@search");
+
+        expectSearchResultItemNameContent({
+          itemNames: [NORMAL_USER_TEST_QUESTION.name, ADMIN_TEST_QUESTION.name],
+        });
+
+        cy.findByTestId("created_by-search-filter").click();
+        popover().within(() => {
+          // remove Robert Tableton from the created_by filter
+          cy.findByTestId("search-user-select-box")
+            .findByText("Robert Tableton")
+            .click();
+          cy.findByText("Apply filters").click();
+        });
+
+        expectSearchResultItemNameContent({
+          itemNames: [ADMIN_TEST_QUESTION.name],
         });
       });
 
       it("should remove created_by filter when `X` is clicked on filter", () => {
-        cy.visit(`/search?q=e&created_by=${NORMAL_USER_ID}`);
+        cy.visit(`/search?q=reviews&created_by=${NORMAL_USER_ID}`);
+
+        expectSearchResultItemNameContent({
+          itemNames: [NORMAL_USER_TEST_QUESTION.name],
+        });
 
         cy.findByTestId("created_by-search-filter").within(() => {
           cy.findByText("Robert Tableton").should("exist");
           cy.findByLabelText("close icon").click();
         });
 
-        // Check that we're getting elements from other users by checking for other types,
-        // since all assets, for the user we're querying are questions
-        cy.findAllByTestId("result-link-info-text").then(
-          $resultTypeDescriptions => {
-            const uniqueTypeDescriptions = new Set(
-              $resultTypeDescriptions.toArray().map(el => el.textContent),
-            );
-            expect(uniqueTypeDescriptions.size).to.be.greaterThan(1);
+        expectSearchResultItemNameContent({
+          itemNames: [
+            NORMAL_USER_TEST_QUESTION.name,
+            ADMIN_TEST_QUESTION.name,
+            REVIEWS_TABLE_NAME,
+          ],
+        });
+      });
+    });
+
+    describe("last_edited_by filter", () => {
+      beforeEach(() => {
+        cy.signInAsAdmin();
```

Comment:
The logic for this `beforeEach` seems heavy to me, open to suggestions for how to change this.